backend/src/main/java/com/google/sps/data/User.java

```diff
@@ -1,31 +1,76 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 package com.google.sps.data;
 
 import com.google.common.base.MoreObjects;
 
 /** A user with saved preferences. */
 public final class User {
 
-  /** Datastore ID */
-  private final long id;
   /** Google username (ldap) */
   private final String username;
+  /** How long user wants to chat */
+  private final int duration;
```

Comment:
This seems better suited for Participant. Participant is a short-lived model that represents a user's single intention to find a match at a specific time. User should only contain data for a user that should live longer than any single attempt to match.

Reply:
I was thinking that some people may prefer 30 min meetings over meetings of any other length, for example. That's the case for me at least. I'll ask about this in our pod meeting.

Reply:
After the discussion sg to keep it. I would amend the comment to mention that this is a preference and not just related to a single matching attempt.